### PR TITLE
issue #520 delete hard-coded file names in bash,bat,src,groovy for regression tests

### PR DESCRIPTION
**Pull request by jyganci**

### CODE REVIEW

File: bash/runcbldemos [hunk 1]
```diff
@@ -5,23 +5,27 @@ set -e              # exit on all errors
 cd $(dirname $0)    # set to directory of script
 cd ..
 
-bash/cblclg zcobol/demo/HELLO $1 $2 $3 $4 $5 $6 $7 $8 $9
+demopath=zcobol/demo
```
Comment (adelosa) [inline bash/runcbldemos:8]: No issue, just curious. Why did you add the demopath variable for the new version?

File: bash/runcmdproc [hunk 2]
```diff
@@ -23,17 +23,37 @@
 # debug flag; 0=no debug, 1=debug
 debug=0
 
-set -e              # exit on all errors
+#set -e              # exit on all errors
```
Comment (adelosa) [inline bash/runcmdproc]: Do you really want to never stop on errors.. I would prefer if you `set +e` when you want to handle return codes and then `set -e` again when you are finished.

**Review by adelosa — DISMISSED** [review]
No issues, just a couple of queries.. I think this has actually simplified the tests and reduced the amount of code which is nice.

**Comment by jyganci** [comment]
1. Usage of demopath (and rtpath). Melvyn used cicspath variables (and
others) in the CICS bat files. It minimizes changes if the value it
represents is used in several places. I've started using the "technique" in
general. Plus, it kind of mimics the "basePath('abc",'def',...)" usage in
the Groovy scripts.

2. Commenting the "set -e" in bash/runcmdproc. My reasoning was so that the
script would have a chance to issue an error/warning message if an invoked
script failed. Rather than sprinkle "set +e", "set -e", put it back the way
it was. Change has been pushed to my fork.

Regards, John Ganci

**Review by adelosa — APPROVED** [review]
Ready to merge!

**Comment by adelosa** [comment]
Merged to main
